zerver/test_external.py

```diff
@@ -224,4 +264,3 @@ def test_change_user(self):
         self.login("hamlet@zulip.com")
         result = self.client.post('/json/users/me/android_gcm_reg_id', {'token':token})
         self.assert_json_success(result)
-
```

Comment:
Im using Atom for it. It seems it does not like any empty newlines at the end of the file. When I added a tab to the line, it stopped removing it